src/tools/batchrun/main.cpp

```diff
@@ -246,19 +127,13 @@ int main(int argc, char* argv[])
         if (not finder.list.empty())
```

Comment:
About : 
```C++
if (not finder.list.empty())
```
is useless :  the loop over found studies won't start if there are no study in the **finder**.

Reply:
Yeah but there will be no clear error message without this condition